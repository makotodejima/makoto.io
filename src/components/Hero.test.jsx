import React from 'react';
import renderer from 'react-test-renderer';
import Hero from './hero';

describe('hero', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Hero />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
