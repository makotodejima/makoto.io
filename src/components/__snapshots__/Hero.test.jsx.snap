// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hero renders correctly 1`] = `
<div
  className="sc-bdVaJa dyBPPB"
>
  <h3>
    MAKOTO DEJIMA
  </h3>
  <h2>
    Front-End Developer / Designer
  </h2>
</div>
`;
